tests/test_application.coffee

```diff
@@ -87,20 +106,42 @@ Space.Error.extend CustomerApp, 'InvalidCustomerName', {
 
 class CustomerApp.Customer extends Space.eventSourcing.Aggregate
 
+  dependencies: {
+    myCommandDependency: 'myCommandDependency'
+    myEventDependency: 'myEventDependency'
+  }
+
   fields: {
     name: String
   }
 
   commandMap: -> {
     'CustomerApp.CreateCustomer': (command) ->
+      @myCommandDependency()
+
       @record new CustomerApp.CustomerCreated {
         sourceId: @getId()
         customerName: command.name
       }
+
+    'CustomerApp.ChangeCustomerName': (command) ->
+      @myCommandDependency()
+
+      @record new CustomerApp.CustomerNameChanged {
+        sourceId: @getId()
+        customerName: command.name
+      }
   }
 
   eventMap: -> {
-    'CustomerApp.CustomerCreated': (event) -> @name = event.customerName
+    'CustomerApp.CustomerCreated': (event) ->
+      @myEventDependency()
```

Comment:
I think none, misunderstood the concept: 

**Aggregate**:
Fired command on new `Aggregate` -> `Router::_setupInitializingMessage` ->  new instance and injection
Fired command on existing `Aggregate` -> `Router::_genericCommandHandler` ->  find instance and injection

events on both paths still have access to dependencies.

**Process**:
Fired command/routed event on new `Process` -> `Router::_setupInitializingMessage` ->  new instance and injection

Routed event on new `Process` -> `Router::_genericEventHandler` ->  find instance and injection (so thats why this only relates to `EXTERNAL EVENT HANDLERS`)

Is that right?